app/models/host.rb

```diff
@@ -88,7 +88,11 @@ def sync_repository(full_name)
         repo.save
         repo.ping_packages_async if repo_changed && repo.persisted?
         repo.sync_extra_details_async if !repo.fork? && repo_changed && repo.persisted? && repo.files_changed?
-        repo.sync_owner
+        begin
+          repo.sync_owner
+        rescue RestClient::NotFound => e
```

Comment:
Can we push the error handling for `sync_owner` up into the individual host classes, as they may not all through the some error class